app/controller/Home.js

```diff
@@ -141,12 +139,11 @@ Ext.define('DevCycleMobile.controller.Home', {
 							params: {
 								os: Ext.os.name + " " + Ext.os.version,
 								device: Ext.os.name,
-								tourId: this.tourInfo.data.tour_id
+								tourId: this.tourInfo.data.tour_id,
+                                id: rider_id
 							},
 							success: function(response){
 								var riderInfo = self.riderStore.first();
-								var decodedResponse = Ext.JSON.decode(response.responseText);
-								rider_id = decodedResponse.rider_id;
```

Comment:
Because, we no longer set the rider_id to whatever the server shoots back at us. We now are setting the rider ID to the Ext.device.Device.uuid.